labman/db/process.py

```diff
@@ -3090,23 +3090,66 @@ def generate_prep_information(self):
                 ('r', 'water_lot_id', 'water_lot'),
             ]
             sql = """
-                SELECT study_id, sample_id, content, run_name, experiment,
-                       fwd_cycles, rev_cycles, principal_investigator,
-                       et.description as sequencer_description,
-                       lpp.epmotion_robot_id as lepmotion_robot_id,
-                       epmotion_tm300_8_tool_id, epmotion_tm50_8_tool_id,
-                       master_mix_id, water_lot_id,
-                       gep.epmotion_robot_id as gepmotion_robot_id,
-                       epmotion_tool_id, kingfisher_robot_id,
-                       extraction_kit_id,
-                       p1.external_id as plate, w1.row_num as row_num,
-                       w1.col_num as col_num,
-                       p2.external_id as primer_composition,
-                       psc.barcode_seq as primer_set_composition,
-                       run_name as run_prefix, sp.sequencer_id as platform_id,
-                       sp.experiment as center_project_name
+                SELECT study_id, sample_id,
+                    -- BARCODE
+                    psc.barcode_seq AS barcode,
+                    -- primer_set_id links to marker_gene_primer_set_id,
+                    -- where we can get the linker/primer
+                    primer_set_id,
+                    -- primer_plate
+                    p2.external_id AS primer_plate,
+                    -- the well_id is a combination of the
+                    --- row/col_num + content
+                    w1.row_num AS row_num, w1.col_num AS col_num, content,
+                    -- where to get the platting name
+                    process.run_personnel_id AS plating,
+                    -- extractionkit_lot
+                    extraction_kit_id,
+                    -- extraction_robot, both the epmotion robot and the
+                    -- kingfisher robot
+                    lpp.epmotion_robot_id AS lepmotion_robot_id,
+                    kingfisher_robot_id,
+                    -- TM1000_8_tool
+                    epmotion_tool_id,
+                    -- mAStermix_lot
+                    mASter_mix_id,
+                    -- water_lot
+                    water_lot_id,
+                    -- processing_robot
+                    gep.epmotion_robot_id AS gepmotion_robot_id,
+                    -- TM300_8_tool
+                    epmotion_tm300_8_tool_id,
+                    -- TM50_8_tool
+                    epmotion_tm50_8_tool_id,
+                    -- sample_plate
+                    p1.external_id AS sample_plate,
+                    -- project_name
+                    study_alias AS project_name,
+                    -- orig_name
+                    sample_id AS orig_name,
+                    -- experiment_design_description
+                    '' AS experiment_design_description,
+                    -- run_center
+                    'UCSDMI' AS run_center,
```

Comment:
Is the center the same if we run on the miseq that sits in our lab space vs. the instruments that are part of the sequencing core? Or those at rady?

Reply:
When I asked "Is this always UCSDMI ?", the answer I got from Gail was "For our lab YES".  I interpreted that to mean that it was true for all prep info sheets created in the lab; should I be asking her a more nuanced question here, and if so, what?

Reply:
Okay. I think it would be useful to ask Gail if UCSDMI encompasses any sequencing instrument associated with the UCSD campus. If yes, than we're good. If no, than more granularity may be warranted

Reply:
Ok, I've put a question out to Gail.  If I learn that we need to make the behavior for this field more nuanced, I'll share that here.